dashboard/src/app/factories/last-factories/last-factories.html

```diff
@@ -17,9 +17,19 @@
     from Codenvy S.A..
 
 -->
-<che-panel che-title="Recent Factories" che-title-icon="material-design icon-ic_widgets_24px">
+<div class="dashboard-last-factories-content">
+
+  <div>
```

Comment:
seems like a duplicate html from last workspaces page but without reusing/sharing the code